### ISSUE TITLE
perPage param ignored

### ISSUE THREAD

**Issue by Benjamin-deVries** [GlobalTechnology/global-registry-java-client#35]
`DEFAULT_PER_PAGE_ENTITIES` should be replaced with `perPage`

https://github.com/GlobalTechnology/global-registry-java-client/blob/601f78565da97be640e605bb5e20e633b26c19ab/api/src/main/java/org/ccci/gto/globalreg/AbstractGlobalRegistryClient.java#L73

**Comment by hlbraddock** [comment]
@Benjamin-deVries 
Is this a dependency for this https://github.com/CruGlobal/giveaem/pull/540 (or the subsequent PR's forthcoming)?

**Comment by Benjamin-deVries** [comment]
Yes, GR is crashing due to memory constraints requiring us to limit down to ~100 per page, as it stands now that is not an option since the perPage param is ignored.

Furthermore, i will be adding this method to allow use of the fields param
```
    public final <T> ResponseList<T> getEntities(final Type<T> type, final int page,
                                                 final int perPage, final String ownedBy,
                                                 final Set<String> fields, final Filter... filters) {
        return this.getEntities(type, page, perPage, fields, ArrayUtil
            .merge(filters, Filter.OWNED_BY.values(ownedBy)));
    }
```
Since the existing one does not allow specification of the page number or the perPage results.
For testing, i am creating a `GrJaxrs20ClientExtender` class in AEM that extends the existing GR client.

**Comment by frett** [comment]
We should probably move this repo to CruGlobal so the respective teams can handle any PRs/changes as appropriate.

**Comment by frett** [comment]
@hlbraddock any complaints with me moving it and then your team taking ownership of it?

**Comment by hlbraddock** [comment]
No complaints at all. Let's do that.

**Comment by frett** [comment]
@hlbraddock it's moved, you may need to have it added to your team in terraform to have proper rights.
